src/olympia/addons/models.py

```diff
@@ -1390,7 +1391,7 @@ def thumbnail_url(self):
             preview = self._all_previews[0]
             return preview.thumbnail_url
         except IndexError:
-            return settings.STATIC_URL + '/img/icons/no-preview.png'
+            return staticfiles_storage.url('img/icons/no-preview.png')
```

Comment:
should probably use this function in `get_default_icon_url` too

Reply:
Yeah I'll do that in https://github.com/mozilla/addons-server/issues/18104